### PR TITLE
Signer tests

### PR DESCRIPTION
**Pull request by markya0616**

### CODE REVIEW

File: crypto/mta/mta.go [hunk 1]
```diff
@@ -32,15 +32,16 @@ var (
 	big0 = big.NewInt(0)
 )
 
-type Mta struct {
+type mta struct {
 	filedOrder *big.Int
```
Comment (cychuang0924) [inline crypto/mta/mta.go]: filedOrder ---> fieldOrder

File: crypto/mta/mta.go [hunk 2]
```diff
@@ -32,15 +32,16 @@ var (
...
 
-	k *big.Int
-	a *big.Int
+	k    *big.Int
+	encK []byte
+	a    *big.Int
 }
 
-func NewMta(filedOrder *big.Int, homoCrypto homo.Crypto) (*Mta, error) {
+func NewMta(filedOrder *big.Int, homoCrypto homo.Crypto) (*mta, error) {
```
Comment (cychuang0924) [inline crypto/mta/mta.go]: filedOrder ---> fieldOrder

File: crypto/mta/mta.go [hunk 3]
```diff
@@ -109,7 +116,7 @@ func (m *Mta) Decrypt(c *big.Int) (*big.Int, error) {
 
 // Compute gets the encrypted k with a random beta
 // alpha = (E(encMessage) * a) + E(beta), where * is the homomorphic multiply amd + is the homomorphic Addition.
```
Comment (cychuang0924) [inline crypto/mta/mta.go]: and ---> and

File: crypto/tss/signer/signer.go [hunk 4]
```diff
@@ -26,22 +27,26 @@ import (
 	"github.com/getamis/sirius/log"
 )
 
+var (
+	ErrInconsistentPeerNum = errors.New("inconsistent peer num")
```
Comment (bun919tw) [inline crypto/tss/signer/signer.go]: This error has been defined in `tss/utils.go`

File: crypto/tss/signer/signer_test.go [hunk 5]
```diff
@@ -15,15 +15,16 @@ package signer
 
 import (
 	"crypto/ecdsa"
+	"crypto/elliptic"
 	"fmt"
 	"math/big"
 	"testing"
 
+	"github.com/getamis/alice/crypto/matrix"
+
```
Comment (bun919tw) [inline crypto/tss/signer/signer_test.go]: redundant empty line

File: crypto/tss/signer/signer.go (hunk 4 above)
Comment (markya0616) [reply crypto/tss/signer/signer.go]: OK

**Pull request merged**